### PR TITLE
[fix](inverted index) update output rowset index meta with input rowset when drop inverted index

### PR DESCRIPTION
**Pull request by Tanya-W**
## Proposed changes

fix problem, example:
if there are two inverted index in table: index_A, index_B
1.  add index_C, the FE's latest index meta is (index_A, index_B, index_C), but now, the stock rowset's index meta in tablet_chema is (index_A, index_B), if no new load or build index, the stock rowset will not create index file for index_C
2. drop index_A, and then update output rowset's tablet_schema with the FE's lasted index meta( index_B, indx_C), and modify output rowsets into meta to instead of input rowsets, the rowset's index meta is changed to (index_B, index_C)
3. drop index_B, the picked input rowset's index meta is (index_B, index_C), but due to index file for index_C is not created, then link file will fail.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by Tanya-W** [comment]
run buildall

File: be/src/olap/task/index_builder.cpp [hunk 1]
```diff
@@ -63,7 +61,9 @@ Status IndexBuilder::update_inverted_index_info() {
         auto input_rs_tablet_schema = input_rowset->tablet_schema();
         output_rs_tablet_schema->copy_from(*input_rs_tablet_schema);
         if (_is_drop_op) {
-            output_rs_tablet_schema->update_indexes_from_thrift(_exist_indexes);
+            for (auto t_inverted_index : _alter_inverted_indexes) {
```
Comment (qidaye) [inline be/src/olap/task/index_builder.cpp:64]: Please add some logic explanation here.

File: fe/fe-core/src/main/java/org/apache/doris/alter/IndexChangeJob.java [hunk 2]
```diff
@@ -290,7 +290,7 @@ protected void runWaitingTxnJob() throws AlterCancelException {
                             partitionId, originIndexId, originTabletId,
                             originSchemaHash, olapTable.getIndexes(),
                             alterInvertedIndexes, originSchemaColumns,
-                            isDropOp, taskSignature);
+                            isDropOp, taskSignature, jobId);
```
Comment (qidaye) [inline fe/fe-core/src/main/java/org/apache/doris/alter/IndexChangeJob.java:293]: Add some explanation for your changes.

**Comment by xiaokang** [comment]
add some description

**Review by xiaokang — DISMISSED** [review]
LGTM

**Comment by Tanya-W** [comment]
run buildall

**Review by qidaye — APPROVED** [review]
LGTM

**Review by xiaokang — APPROVED** [review]
LGTM

**Pull request merged**